client/my-sites/domains/domain-management/edit/domain-only-cta.jsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import React, { useCallback } from 'react';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+import { useSelector } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import { isExpiringSoon } from 'lib/domains/utils';
+import {
+	domainManagementNameServers,
+	domainManagementTransferToOtherSite,
+} from 'my-sites/domains/paths';
+import getCurrentRoute from 'state/selectors/get-current-route';
+
+const DomainOnlyCta = ( { domain, selectedSiteSlug, translate } ) => {
```

Comment:
Can we rename this to something else? `DomainOnlyCta` is not really descriptive.

Reply:
Naming is a very complex - isn't it. I just changed it to `DomainOnlyNotice`.

Reply:
Thanks for the ping! Let's keep this as parked because in that other case, we're using `status-pending` for activating which I was hoping to treat differently.